packages/graphback-runtime-mongodb/src/MongoDBDataProvider.ts

```diff
@@ -50,18 +50,20 @@ export class MongoDBDataProvider<Type = any> implements GraphbackDataProvider<Ty
 
   public async update(data: Type, context: GraphbackContext): Promise<Type> {
     const { idField } = getDatabaseArguments(this.tableMap, data);
+    const updateSets = Object.assign({}, data as any);
+    delete updateSets.id;
```

Comment:
`getDatabaseArguments` will return an ID field different to `id` if the `@id` annotation is different. So should this be deleting that key instead?